index.html

```diff
@@ -160,6 +160,28 @@ <h2 class="about-header">About me</h2>
             </ul>
         </div> 
         <!-- work section ends       -->
+        
+        <div class="form-start">
+        <h2 class="form-header">I'm always interested in hearing about your new projects, so if you'd like to chat please get in</h2>
+        <form>
+          <label for="full-name"></label>
+          <input type="text" id="full-name" name="full-name" maxlength="30" placeholder="Full name" required>
+          <label for="email"></label>
+          <input type="email" id="email" name="email" placeholder="Email address" required>
+          <label for="message"></label>
+          <textarea id="message" name="message" maxlength="500" required>Enter text here</textarea>
+          <input type="submit" value="Get in touch" class="submit">
+        </form>
+            <hr class="seperator">
+            <div class="footer">
```

Comment:
[ ] [OPTIONAL] Kindly ensure to wrap the form section inside the `section` HTML tag. This is to ensure that our portfolio is [Semantically](https://www.w3schools.com/html/html5_semantic_elements.asp) correct and also increase its accessibility to screen readers ☺️.